examples/docs/gatsby-node.js

```diff
@@ -70,11 +69,16 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
 
   if (node.internal.type === `Mdx`) {
     const parent = getNode(node.parent);
-    const value = parent.relativePath.replace(parent.ext, "");
+    let value = parent.relativePath.replace(parent.ext, "");
+
+    if (value === "getting-started") {
```

Comment:
seems... weird? Why is `getting-started` special-cased here?

Reply:
For the homepage. I'm good with having a more classical home page.